### PR TITLE
Update Capistrano

### PR DESCRIPTION
**Pull request by Pontus4**
The previous versions were very old (2015 and 2016). 

The update also resolves Ruby 2.7 warnings like https://github.com/twingly/blantyre/issues/101.

### CODE REVIEW

**Comment by Pontus4** [comment]
twingly/test has been deployed successfully with these changes 👍

**Comment by walro** [comment]
What was the exact error message?

**Comment by Pontus4** [comment]
> What was the exact error message?

One error led to many, but this was the first one:

```
00:08 deploy:export_service
      01 sudo systemctl stop twingly-test.target
    ✔ 01 deploy@tuor.sth.twingly.network 0.077s
    ✔ 01 deploy@haldir.sth.twingly.network 0.084s
    ✔ 01 deploy@fili.sth.twingly.network 0.077s
      02 sudo /usr/local/bin/chruby-exec 2.7 -- /home/deploy/twingly-test/shared/bin/foreman export systemd /etc/systemd/system -a twingly-test -u `whoami` -l /home/deploy/twingly-…
      02 /opt/rubies/ruby-2.7.1/lib/ruby/2.7.0/bundler/definition.rb:32:in `build': /home/deploy/twingly-test/releases/20201123133912/Gemfile not found (Bundler::GemfileNotFound)
```

**Comment by walro** [comment]
Regarding the fix, isn't it possible to step version by version for, I suspect, Capistrano Bundler? To me it seems like we don't know why it broke down, but if we can pinpoint the exact release it should be easier to figure it out what happened.

A first step could be to try out 1.6.0.

That said, I think the change is good either way as it sets the "variable" where needed instead of the more global version we had before :)

File: capistrano-twingly.gemspec [hunk 1]
```diff
@@ -17,8 +17,8 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "capistrano", "~> 3.6.0"
-  spec.add_dependency "capistrano-bundler", "1.1.4"
+  spec.add_dependency "capistrano", "~> 3.14"
+  spec.add_dependency "capistrano-bundler", "2.0"
```
Comment (walro) [inline capistrano-twingly.gemspec:21]: What does this resolve to, just 2.0.0? I think we should at least go with "~> 2.0"

Comment (Pontus4) [reply capistrano-twingly.gemspec:21]: Oh, that must have been because of the previous static version. I agree we should add the "~>"!

**Comment by Pontus4** [comment]
> Regarding the fix, isn't it possible to step version by version for, I suspect, Capistrano Bundler?

Okay, it works for v1.2.0 but not for v1.3.0, which must be because of this change: https://github.com/capistrano/bundler/compare/v1.2.0...v1.3.0#diff-9d850614d36625acd080ece560b529927117cea9a22c20223e791219e51db489R77.

I guess `set :bundle_binstubs, -> { shared_path.join('bin') }` is overwritten by `nil` when it is defined in the namespace instead of in the task.

**Comment by Pontus4** [comment]
> I guess set :bundle_binstubs, -> { shared_path.join('bin') } is overwritten by nil when it is defined in the namespace instead of in the task.

We have now confirmed that this is the case.

**Comment by walro** [comment]
Cool, maybe the commit message can be re-vised a bit with the new found information and then I think we're good to go

**Comment by roback** [comment]
There was probably not much thinking behind the original placement of the `set :bundle_binstubs, -> { shared_path.join('bin') }` line in 50c4aeefc479797f3369670d31f2c703779f120d´ 😄. I think I just placed it there because it happened to work. The documentation for capistrano-bundler isn't the best...

**Comment by Pontus4** [comment]
> Cool, maybe the commit message can be revised a bit with the new found information and then I think we're good to go

Updated in f3f323c 👍

File: capistrano-twingly.gemspec (hunk 1 above)
Comment (walro) [reply capistrano-twingly.gemspec:21]: I noticed the comment was resolved, but no change :)

Comment (Pontus4) [reply capistrano-twingly.gemspec]: 🤦 

Here you go: e099d55 😃

**Review by walro — APPROVED** [review]
*caps his hands*

**Pull request merged**